src/core/Akka.Cluster/ClusterDaemon.cs

```diff
@@ -285,11 +285,6 @@ internal class JoinSeenNode
         /// <inheritdoc cref="JoinSeenNode"/>
         internal class InitJoin : IClusterMessage, IDeadLetterSuppression
         {
-            /// <inheritdoc/>
-            public override bool Equals(object obj)
```

Comment:
This `Equals` method is not necessary and was generating a build warning. Removed it.

Reply:
Welp, tests are failing now that I've removed this - guess I'm going to have to add it back cc @Arkatufus

Reply:
This has been fixed.